docs/tctl.md

```diff
@@ -269,9 +269,10 @@ You can reset to some predefined event types:
 ./tctl workflow reset -w <wid> -r <rid> --reset_type <reset_type> --reason "some_reason"
 ```
 
-- FirstDecisionCompleted: reset to the beginning of the history.
-- LastDecisionCompleted: reset to the end of the history.
+- FirstWorkflowTask: reset to the beginning of the history.
+- LastWorkflowTask: reset to the end of the history.
 - LastContinuedAsNew: reset to the end of the history for the previous run.
+- BadBinary: <>.
```

Comment:
@mastermanu, @cullywakelin do you know what this resets to?

Reply:
it resets the workflow to the point where a workflowtask executed with the version of the bad binary and then starts from there